CONTRIBUTING.md

```diff
@@ -20,22 +20,26 @@ The Rails Girls Summer of Code website is built using [Jekyll](https://jekyllrb.
 "Forking" basically just means "making a copy"; this allows you to copy our `summer-of-code` repo (found [here](https://github.com/rails-girls-summer-of-code/summer-of-code)) without affecting it. Follow the instructions on forking a repo and cloning your fork [in the GitHub docs](https://help.github.com/articles/fork-a-repo/). When forking, you might be asked where you want to fork the repository to; select **your own Github profile**.  
 Once you're all set up, you'll have to bundle the gems using bundler and start jekyll:
 
+
 ```
 gem install bundler
 bundle install
 jekyll serve --watch
 
 ```
 
+#### Note:Ruby Dependencies
+Both .ruby-version and Gemfile handle the Ruby version (2.1.5) for this application. Users with or without a RVM (Ruby Versioning Manager) should be able to interact and contribute to the code base. If running an older local Ruby version on your machine, you may need to `bundle update` to reset Gemfile.lock. 
```

Comment:
two comments about this section:  
- I don't think it's necessary to specify the exact Ruby version here, you just remove the `(2.1.5)` — that way we could avoid the README becoming accidentally outdated the next time we update our Ruby version. What do you think?
- I'd suggest to make the bit about Ruby versioning more general, something along the lines of:
"User who don't use a tool to manage Ruby versions (such as [RVM](https://rvm.io/) or [rbenv](https://github.com/rbenv/rbenv)) should be able to [...]". This way we avoid to specifically show preference about a single tool.